### ISSUE TITLE
Increasing the proportion of Iron

### ISSUE THREAD

**Issue by LBrinkmann** [nthByte-LLC/iron-craft#31]
It is currently very hard to find iron. Increasing the rate of iron underground would be great.

**Comment by its-c10** [comment]
Increased within the tutorial. Also removed the diamonds from the mining area, as it was unnecessary. Working on artificially increasing the iron within the actual world.

**Comment by LBrinkmann** [comment]
Yes, in the tutorial it was actually ok, but in the actual world, I was still struggling to find iron in time.
